src/gluonts/dataset/common.py

```diff
@@ -288,58 +273,22 @@ class Config:
         arbitrary_types_allowed = True
 
     freq: Union[str, pd.DateOffset]
+    use_timestamp: bool = False
     name: str = FieldName.START
-    tz_strategy: TimeZoneStrategy = TimeZoneStrategy.error
 
     def __call__(self, data: DataEntry) -> DataEntry:
         try:
-            timestamp = ProcessStartField.process(data[self.name], self.freq)
+            if self.use_timestamp:
+                data[self.name] = pd.Timestamp(data[self.name])
+            else:
+                data[self.name] = pd.Period(data[self.name], self.freq)
```

Comment:
Why do we need this option?

Reply:
For point-processes datasets, where `start` needs to be a `Timestamp`. These also have an `end` field which needs to be a timestamp, which we don't handle.

It's a hack.